internal/lib/lib.go

```diff
@@ -1913,3 +1913,27 @@ func IsChanClosed(ch <-chan struct{}) bool {
 	}
 	return false
 }
+
+func GetIPFromNode(node *v1.Node) (string, string) {
+	var nodeV4, nodeV6 string
+	nodeAddrs := node.Status.Addresses
+	for _, addr := range nodeAddrs {
+		if addr.Type == "InternalIP" {
```

Comment:
`InternalIP` can be changed to the constant `corev1.NodeInternalIP`

Reply:
updated to use the string constant